src/components/Button.jsx

```diff
@@ -23,7 +23,7 @@ const buttonVariants = cva(
   },
 );
 
-function Button({ className, icon = false, icon_pos, size, text, variant, ...props }) {
+function Button({ className, icon = false, icon_pos, size, variant, ...props }) {
```

Comment:
Also let's change `icon_pos` to `iconPos` in the scope of this task to make it look like JS.